integration/manifests/cleaner/cleaner-for-calico.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: danm-cleaner
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: caas:danm-cleaner
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - watch
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - endpoints
+  verbs:
+  - create
+  - update
+  - get
+- apiGroups:
+  - danm.k8s.io
+  resources:
+  - danmnets
+  - danmeps
+  - tenantnetworks
+  - clusternetworks
+  verbs: [ "*" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: caas:danm-cleaner
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: caas:danm-cleaner
+subjects:
+- kind: ServiceAccount
+  name: danm-cleaner
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: danm-cleaner
+  namespace: kube-system
+  labels:
+    danm: cleaner
+spec:
+  selector:
+    matchLabels:
+     danm: cleaner
+  template:
+    metadata:
+      name: danm-cleaner
+      labels:
+        danm: cleaner
+    spec:
+      serviceAccountName: danm-cleaner
+      containers:
+      - name: danm-cleaner
+        image: cleaner
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+        - name: calico-config
+          mountPath: /etc/calico/calicoctl.cfg
+          readOnly: true
+        - name: calicoctl
+          mountPath: /usr/local/sbin/calicoctl
+          readOnly: true
+        - name: etcd-certs
```

Comment:
do we really need ETCD backend certs to communicate to Calico control plane?

Reply:
calicoctl was complaining about them when I was testing it

Reply:
could you share?

Reply:
Here is a sample calico cleanup log output when no etcd certificatea are provided for calicoctl:
```
2020/12/02 19:19:17 Delete calico managed IP:/usr/local/sbin/calicoctl ipam release --ip=192.168.95.102
2020/12/02 19:19:17 WARNING: Danmep:fd41e9de-1fcb-4da5-a4d4-cb527eaa4f2c in namespace:kube-systemcould not be cleaned because of error:could not release calico IP 192.168.95.102/32, because: exit status 1 | output: could not initialize etcdv3 client: open /etc/etcd/ssl/etcd-client.pem: no such file or directory
```

Reply:
as above
CERTs should be Secrets at least tho, not ConfigMaps, otherwise can live with it for now

Reply:
I updated it to use Secrets